File: 1er Trimestre/EjercicioAlgoritmo2_Hoja3(real)/ejercicio3.py
"""Se desea codificar en Java un programa que nos permita
introducir por teclado una palabra, seguidamente el programa
deberá indicarnos si  la palabra introducida es un palíndromo o no.
Por ejemplo:
Palabra introducida:  Azulejo  Resultado:  No es palíndromo.
Palabra introducida:  Reconocer Resultado:  Es palíndromo. """

palabra = str(input("Introduce una palabra"))
suma = ''
suma = (suma + palabra[::-1])
print(suma)
if suma == palabra:
    print("Tu palabra es palíndroma")
else:
    print("Tu palabra no es palíndroma")

File: ProgramasVarios/src/ejercicios002_2/Ejercicio3.java
package ejercicios002_2;
import javax.swing.*;
public class Ejercicio3 {

public static void main (String[] args) {
	/*Deseamos codificar un programa en Java que nos permita
	 introducir por teclado un número entero y positivo,
	 posteriormente el programa invertirá el número introducido,
	 mostrando el resultado en la pantalla. 
	 Por ejemplo, número introducido 1526984, número invertido 4896251. */
	
int num, cont, i, cuenta;

num = Integer.parseInt(JOptionPane.showInputDialog(null, "Inserta un número"));
i = num;
	do {
		i = i%10;
		for (cuenta = 0; cuenta >= num; cuenta++);
			JOptionPane.showMessageDialog(null, cuenta);
	}while(num >= 0);
}
}
